lib/controller.js

```diff
@@ -122,6 +124,35 @@ export class AdvancedOpenFileController {
         this.openPath(path);
     }
 
+    openNewFile(path) {
+        if (config.get('newFileConfirmation')) {
+            if (!this.isWaitingForNewFileConfirmation) {
+                this.isWaitingForNewFileConfirmation = true;
+                this.view.showNewFileConfirmation();
+                atom.beep();
```

Comment:
IMO beeps are for when the user is prevented from doing something, but in this case we're confirming their intention, and shouldn't beep.